### PR TITLE
Badge Component

### PR DESCRIPTION
**Pull request by m26unkwn**
Variant
1. DOT Badge eg online,offline,dnd
2. Num Badge

### CODE REVIEW

**Comment by ashishkatiyar2404** [comment]
Your documentation part is really nice(reading view). including UI your documentation also looks good. 
I have seen you use badges on diff positions that's good.
You have used carbon which is also matching to your UI again which is awesome.

**Comment by m26unkwn** [comment]
Thanks @ashishkatiyar2404 Means a lot brother.

File: styles/badge.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  width: var(--badge-dot-width);
+  height: var(--badge-dot-height);
+}
+
+.badge-pos-bottom-right {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  border-radius: 50%;
+  border: 2px solid #ffff;
```
Comment (aiman-ansari) [inline styles/badge.css:31]: It would be good if you use color variable instead of direct value.

**Review by aiman-ansari — CHANGES_REQUESTED** [review]
UI looks so clean also color combination is so good. Badge position looks perfect in all variation.
Just change that color variable part

File: styles/badge.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
```
Comment (dhimanaishwarya) [inline styles/badge.css:32]: you can add a variable for #ffff

File: components/badge.html [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+  </head>
+  <body>
+    <main class="main-wrapper flex flex-col">
+      <!-- navbar starts -->
+      <nav class="nav-wrapper">
+        <div class="container nav-inner-wrapper flex ai-center jc-between">
+          <h1 class="nav-logo">LightUI</h1>
+          <div class="nav-items flex flex-centered">
+            <a href="#!">Docs</a>
+            <a href="#!">Github</a>
```
Comment (dhimanaishwarya) [inline components/badge.html:22]: update the links

File: styles/badge.css (hunk 1 above)
Comment (m26unkwn) [reply styles/badge.css:31]: thanks, Aiman for reminding me about this.

File: styles/badge.css (hunk 2 above)
Comment (m26unkwn) [reply styles/badge.css:32]: will do this. Thanks Aishwarya

File: components/badge.html (hunk 3 above)
Comment (m26unkwn) [reply components/badge.html:22]: will do it when it will complete thanks.

**Review by j24m — APPROVED** [review]

**Pull request merged**